fastai/text/learner.py

```diff
@@ -84,18 +99,29 @@ def load_model_text(file, model, opt, with_opt=None, device=None, strict=True):
 @delegates(Learner.__init__)
 class TextLearner(Learner):
     "Basic class for a `Learner` in NLP."
-    def __init__(self, dls, model, alpha=2., beta=1., moms=(0.8,0.7,0.8), **kwargs):
+    def __init__(self,
+        dls:DataLoaders, # Text `DataLoaders`
+        model, # Pretrained Model Checkpoint or Custom Model used to train
+        alpha:float=2., # Param for `RNNRegularizer`
```

Comment:
Are `alpha` and `beta` actually used in TextLearner?

Reply:
Yes. TextLearner adds [RNNCallback](https://docs.fast.ai/callback.rnn.html#rnn_cbs) as part constructor initialization. alpha and beta are the params used by the [RNNRegularizer](https://github.com/fastai/fastai/blob/master/fastai/callback/rnn.py#L25)

Reply:
I'm curious if it actually is using them currently though.  Wouldn't we need to pass alpha and beta to rnn_cbs for those values to be respected?  I think the current code would use the default values of `rnn_cbs` rather than `alpha` and `beta` that we are passing in

Reply:
![image](https://user-images.githubusercontent.com/7451178/156895591-10024d2f-3f1d-4ba9-88a5-6c8a62851154.png)

Just did a quick test of this.  If alpha and beta are both 0 (default in rnn_cbs) the RNNRegularizer cb isn't added.  So this shows that pretty clearly unless I'm missing something.  I will add an issue on fastai main branch though since that is a code change

Reply:
reviewing it again. It actually doesn't use it. uses the default values passed. May need a separate pr to fastai. Nice catch. :)

Reply:
https://github.com/fastai/fastai/issues/3605